Assets/HotmaxEditorVR/builder/prop/scaleControl.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class scaleControl : MonoBehaviour
+{
+    private Vector3 growRate = new Vector3(.01f, .01f, .01f);
+
+    private float initialYPos;
+    private float currentYPos;
+
+    private Vector3 rayOrigin;
+    private Vector3 rayDirection;
+
+    private void OnEnable()
+    {
+        inputManager.trackedController2.TriggerClicked += triggerClicked;
+        init.scaleController.SetActive(true);
+    }
+
+    private void OnDisable()
+    {
+        inputManager.trackedController2.TriggerClicked -= triggerClicked;
+        init.scaleController.SetActive(false);
+    }
+
+    void triggerClicked(object sender, ClickedEventArgs e)
+    {
+        getInitialHitYPosition();
+    }
+
+    void Update()
+    {
+        setScaleControllerToFacePlayer();
+
+        if (inputManager.trackedController2.triggerPressed)
+        {
+            scaleObject();
+        }
+    }
+
+    void setScaleControllerToFacePlayer()
+    {
+        Transform scaleControllerHolder = init.scaleController.transform.parent.transform;
```

Comment:
same with these.. create local variables that hold init.<whatever> at top of file